### DISCUSSION TITLE
Strange Value display when using ColorPicker on Skia

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#13756] [Q&A]
Sometimes I get this strange label that shows up in the left-hand corner when picking a color. I have seen this on Skia on GTK+ on Windows, and on Linux Framebuffer. Where is it coming from, and more importantly, how can I disable it? It doesn't show up all the time, I don't have a reliable way to reproduce it.

![image](https://github.com/unoplatform/uno/assets/407941/91eef1d6-57de-47fe-9d8b-19973863cdac)

**Comment by jeromelaban** [comment]
Let's track in https://github.com/unoplatform/uno/issues/13795
